samples/Calculator/windows/Calculator/Calculator.vcxproj

```diff
@@ -5,18 +5,18 @@
     <CppWinRTOptimized>true</CppWinRTOptimized>
     <CppWinRTRootNamespaceAutoMerge>true</CppWinRTRootNamespaceAutoMerge>
     <MinimalCoreWin>true</MinimalCoreWin>
-    <ProjectGuid>{0a214f22-389d-4fb1-97b0-7ea8a092fcd8}</ProjectGuid>
+    <ProjectGuid>{3e279328-1283-46d4-a5d4-44e691939b29}</ProjectGuid>

```

Comment:
Hmm, so basically we deleted the original project entirely and created a new one? Should we keep the same ProjectGuid as the original project?

Reply:
I guess in a real-world scenario, esp. with an existing app that has already been published to the store, even if you the dev never made any changes to the native code, you wouldn't want to obliterate any "packaging" changes like adding your icons, store metadata, signing certs, etc.

But in this case, this sample existed as a kind of prototype sample before we announced MVP. This PR updates the sample to be a new project using the first stable version we're telling people to really use.

Future updates to the samples, (to new stable versions) should probably not just obliterate everything.